lib/models/onboarding/onboarding_financial_details_attributes.dart

```diff
@@ -45,3 +45,44 @@ enum OnboardingMaritalStatus { notMarried, married, divorced, widowed, preferNot
 enum OnboardingLivingSituation { own, rent, parents }
 
 enum OnboardingOccupationalStatus { employed, unemployed, apprentice, retired, student }
+
```

Comment:
I think that it would be easier to get the string values from these enums if we use an `Extension`, so when we can have something like `OnboardingMaritalStatus.notMarried.getValue()` which can return the needed string value